src/api/test-controller/index.js

```diff
@@ -213,6 +218,51 @@ export default class TestController {
         return this._enqueueCommand(DispatchEventCommand, { selector, eventName, options, relatedTarget: options.relatedTarget });
     }
 
+    _getUrls (args) {
+        const [, urls] = args;
+
+        if (typeof urls === 'string')
+            return castArray(urls);
+
+        return Array.isArray(urls) && typeof urls[0] === 'string' ? flattenDeep(urls) : [];
+    }
+
+    _normalizeCookies (cookies, isSetCommand = false) {
+        const normalizedCookies = [];
+
+        flattenDeep(castArray(cookies)).forEach(cookie => {
+            if (isSetCommand && !cookie.name && typeof cookie === 'object')
+                Object.entries(cookie).forEach(([name, value]) => normalizedCookies.push({ name, value }));
+            else if (!isSetCommand && typeof cookie === 'string')
+                normalizedCookies.push({ name: cookie });
+            else
+                normalizedCookies.push(cookie);
+        });
+
+        return normalizedCookies;
+    }
+
+    [delegatedAPI(GetCookiesCommand.methodName)] (...args) {
+        const urls    = this._getUrls(args);
```

Comment:
I propose to pack the `_getUrls` and `_noramalizeCookies` methods to the single `_prepareCookieArguments` and cover it in the server tests.